### ISSUE TITLE
The contents of the file are not compared. Compare only file names.

### ISSUE THREAD

**Issue by 57382** [WinMerge/winmerge#997]
![image](https://user-images.githubusercontent.com/81396211/136544751-53955ff0-bfee-451a-a9d4-257acd3a0875.png)

https://freecommander.com/fchelpxe/en/SettingsCompareFolders.html
This feature is particularly important. This is what similar software is missing.

**Comment by 57382** [comment]
This also applies to folders that compare two directories. Compare folder names only. The ability to ignore or not ignore subfolders.

**Comment by Gitoffthelawn** [comment]
> https://freecommander.com/fchelpxe/en/SettingsCompareFolders.html This feature is particularly important. This is what similar software is missing.

I'm confused by this request.  Are you saying that FreeCommander doesn't ever actually compare the contents of the files, and you want WinMerge to have an option to do the same?

**Comment by 57382** [comment]
> 

No. FreeCommander did a great job. I wish WinMerge had an option: not to compare file contents.

**Comment by 57382** [comment]
> > https://freecommander.com/fchelpxe/en/SettingsCompareFolders.html This feature is particularly important. This is what similar software is missing.
> 
> I'm confused by this request. Are you saying that FreeCommander doesn't ever actually compare the contents of the files, and you want WinMerge to have an option to do the same?

No. FreeCommander did a great job. I wish WinMerge had an option: not to compare file contents.

**Comment by gekigek99** [comment]
I think being able to compare filenames only is fundamental.

Is probably the simplest implementation out of all the possible ones (already imolemented) and therefore is needed.
